net-p2p/resilio-sync/resilio-sync-2.6.1.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+BASE_URI="http://linux-packages.resilio.com/${PN}/deb/pool/non-free/r/${PN}/${PN}_${PV}-1_@arch@.deb"
+
+inherit pax-utils readme.gentoo-r1 systemd tmpfiles unpacker user
+
+NAME="rslsync"
+QA_PREBUILT="usr/bin/${NAME}"
```

Comment:
Please move this below basic ebuild variables. Keeping variables in ~same order in ebuilds makes it easier to find stuff.

Reply:
That is really, really weird. Could you try again? I don't see how variable definition order would matter here.

Reply:
That is because:
QA_PREBUILT="usr/bin/${NAME}"
NAME="rslsync"

Using NAME really was not a good idea. Removed.